java/src/lib.rs

```diff
@@ -168,8 +168,12 @@ fn array_to_java_array<'local>(
     Ok(items.into())
 }
 
-#[no_mangle]
-pub extern "system" fn Java_glide_ffi_resolvers_GlideValueResolver_valueFromPointer<'local>(
+/// # Safety
+/// All FFI functions are considered `unsafe` by default.
```

Comment:
Where does it say that all extern functions should be unsafe? I think this should be marked `unsafe` because we take an arbitrary raw pointer and convert it to `Box` using `Box::from_raw`, but this comment doesn't have anything to do with that. The safety documentation should explain what pre-conditions the caller needs to ensure before calling this function.

Reply:
Applies throughout.

Reply:
Fixed